src/main/scala/org/dia/algorithms/mcc/GTGRunner.scala

```diff
@@ -348,11 +350,20 @@ class GTGRunner(val masterURL: String,
     val subgraphs = sc.sparkContext.parallelize(edgeListRDDIndexed.toSeq)
       .map(MCCOps.mapEdgesToBuckets(_, maxParitionSize, buckets))
       .groupByKey()
-    val subgraphsFound = MCCOps.findSubgraphsIteratively(subgraphs, 1, maxParitionSize, minGraphLength, sc.sparkContext)
+    val subgraphsFound = MCCOps.findSubgraphsIteratively(subgraphs, 1, maxParitionSize,
```

Comment:
Ack'ed to 1. and 2. @rahulpalamuttam. The updated push here addresses 1.
For 2. I agree a new PR should be made. Note that associated with 2. the warning msg: 
'WARN TaskSetManager: Stage 8 contains a task of very large size (290 KB)' returns. So we really want to refactor that for the larger jobs.